.github/sync/RELEASE.md

```diff
@@ -61,21 +61,31 @@ Placeholder for `{{ repo.name }} YY.M.0` release.
     <!-- link any other feedstock PRs here -->
 - [ ] Hand off to the Anaconda packaging team
 - [ ] Announce release
-    - [ ] Create release [announcement draft][announcement]
-    - [ ] Discourse
-    - [ ] Twitter
-    - [ ] Matrix
+    - Blog Post (optional)
+        - [ ] conda.org (link to pull request)
+    - Long form
+        - [ ] Create release [announcement draft](https://github.com/conda/communications)
+        - [ ] [Discourse](https://conda.discourse.group/)
+        - [ ] Matrix
+    - Summary
+        - [ ] Twitter
 ```
-
 </details>
 
 
-## 2. Ensure `rever.xsh` and `news/TEMPLATE` are up to date.
+> **Note**
+> The new release branch should adhere to the naming convention of `YY.M.x`
```

Comment:
```suggestion
> The new release branch should adhere to the naming convention of `YY.M.x`.
```

Totally understand if we feel leaving the period out is less confusing for formatting reasons. I forget, do the single backticks do italics or code formatting? If the latter, the period won't be a problem, IMO.